### ISSUE TITLE
Regular update needed

### ISSUE THREAD

**Issue by zb-markus** [klokantech/kartenportal-tickets#229]
We were not able to find some maps in Kartensuche that have been catalogued since mid-January. Here 3 examples:

http://aleph.unibas.ch/F/?local_base=DSV01&con_lng=GER&func=find-b&find_code=SYS&request=006215130
http://www.swissbib.ch/TouchPoint/perma.do?q=0%3D%22225988879%22+IN+%5B4%5D&v=nose&l=de

http://aleph.unibas.ch/F/?local_base=DSV01&con_lng=GER&func=find-b&find_code=SYS&request=006215129
http://www.swissbib.ch/TouchPoint/perma.do?q=0%3D%22225988224%22+IN+%5B4%5D&v=nose&l=de

http://opac.nebis.ch/F/?local_base=NEBIS&CON_LNG=GER&func=find-b&find_code=SYS&request=010047400
http://www.swissbib.ch/TouchPoint/perma.do?q=0%3D%22225753596%22+IN+%5B4%5D&v=nose&l=de

However, another map catalogued in mid-January has been found. The picture is not clear, however, it seems that quite a lot of recent records are missing. Please make reliable daily updates possible again.

**Comment by zb-markus** [comment]
Interestingly, title words from the first and second examples above are autocompleted within the test version. If you type "Centrals" into the title attribute field, then the second proposal of the autocomplete function is the correct "Centralschule". But the record (i.e. first example above) does not appear within the Trefferliste. 

Likewise, type "Wahlen", then the second proposal of the autocomplete function is the correct "Wahlenbach". Here, too, the record (i.e. second example above) is missing within the Trefferliste. Note that this is especially significant because the missing record is the only one within Swissbib that has "Wahlenbach" in its title (there is another useless map record in te Swissbib test version without coordinates). 

In conclusion, it seems that the update as such is probably working regularly. However, some records are not visible while at least their titles are indexed.

**Comment by klokan** [comment]
We are investigating the issue.

**Comment by klokan** [comment]
This should be fixed since the end of the last week. The data were correctly synchronised, but the MapRank Server has not been restarted properly after the update. Therefore the difference in the autocomplete and main MapRank.

@zb-markus could you please verify?

**Comment by zb-markus** [comment]
OK, the above-mentioned examples are now retrievable in the Kartensuche. We will test more during the next days. 

I cannot say whether there is the same bug behind, but the figures in the library list are still not updated. The figures given are the same as in beginning of January 2014 (examples: ZB Zürich 24889, UB Basel Hauptbibliothek 4642).

**Comment by keosak** [comment]
You are right, there was a problem with updating the library figures due to a broken software dependency. It is fixed now, so the numbers should be up to date.
